src/Modules/CalcDefence.lua

```diff
@@ -2328,14 +2329,15 @@ function calcs.defence(env, actor)
 
 		local function damageMultiplierForType(damage, ofType)
```

Comment:
This is probably going to be inaccurate with armour as enemyDamageMult should also apply to armour's damage value

Reply:
Yea i know but then this function isnt always used and I was unsure how to also apply it to armour for other code paths so I just decided to skip it unless I figure out how